testing/litespeed/APKBUILD

```diff
@@ -56,8 +56,7 @@ package() {
 		"$pkgdir"/var/log || return 1
 
 	# remove trash
-	rm -fr "$pkgdir"/$_pkghome/php* \
```

Comment:
any idea what stored here? 
Guess lsapi for php https://openlitespeed.org/kb/installation-from-source-code/#Install_PHP

needs @vakartel to approve

Reply:
There is something wrong with this package, the above only fixed some of it so I need to take another look